comps/comps-foreman-rhel7.xml

```diff
@@ -34,6 +34,7 @@
       <packagereq type="default">foreman-service</packagereq>
       <packagereq type="default">foreman-telemetry</packagereq>
       <packagereq type="default">foreman-vmware</packagereq>
+      <packagereq type="default">nodejs-downshift</packagereq>
```

Comment:
You need to do the same thing for EL8. The script doesn't handle that. Same thing for the package manifest.

Reply:
updated, but not sure where I should manually update the package manifest?